orga/tests/integration/components/edit-student-number-modal-test.js

```diff
@@ -65,7 +64,7 @@ module('Integration | Component | edit-student-number-modal', function(hooks) {
 
       test('should have the update button enable', async function(assert) {
         // when
-        await fillInByLabel('Nouveau numéro étudiant', this.student.studentNumber);
```

Comment:
**question** : pourquoi remplacer `fillInByLabel('Nouveau numéro étudiant', this.student.studentNumber)` par `fillIn('#editStudentNumber', this.student.studentNumber)` ? Cela ne fonctionnait pas ?

Je pense que la version d'avant est plus robuste et donc plus maintenable, car cette ligne n'est pas censée faire échouer le test si on ne se base pas sur l'id du champ.

Reply:
Je l'ai changé car il fonctionnait plus.. il me mets une erreur comme qua il a trouvé le label mail pas le controller associé, j'ai regardé avec le debugger et en effet , le champ d'input reste vide et donc les testes ne passent pas.

Je ne sais pas pourquoi mais en remplacement `fillInByLabel` par `fillIn` ou `typeIn` ça passe.

Reply:
Corrigé: on utilise maintenant `fillInByLabel`